### PR TITLE
Port PR #705 changes

### PR DESCRIPTION
**Pull request by AmintorDusko**
**Context:** Port changes from PR #705 

**Description of the Change:**

**Benefits:**

**Possible Drawbacks:**

**Related GitHub Issues:**

### CODE REVIEW

File: pennylane_lightning/lightning_qubit/lightning_qubit.py [hunk 1]
```diff
@@ -708,7 +739,9 @@ def execute_and_compute_vjp(
...
         batch_obs = execution_config.device_options.get("batch_obs", self._batch_obs)
         results = tuple(
-            simulate_and_vjp(circuit, cots, self._statevector, batch_obs=batch_obs)
+            simulate_and_vjp(
+                circuit, cots, self._statevector, batch_obs=batch_obs, wire_map=self._wire_map
+            )
             for circuit, cots in zip(circuits, cotangents)
         )
-        return tuple(zip(*results))
+        return tuple(zip(*results))
```
[the remark this answers is not in the document]
Comment (AmintorDusko) [reply pennylane_lightning/lightning_qubit/lightning_qubit.py:747]: Thanks!

**Review by vincentmr — APPROVED** [review]
If would be nice to hit those patch lines, but I think we can live with it for now. Thanks @albi3ro .

**Comment by vincentmr** [comment]
Oh, and thanks @AmintorDusko for the PR :)

**Comment by AmintorDusko** [comment]
Yes, @vincentmr, but I think the coverage will increase with @mlxd's PR.
Unfortunately, we are short on time.

**Pull request merged**